Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/Setup.cs

```diff
@@ -19,19 +27,29 @@ protected override IMvxIoCProvider CreateIocProvider()
         {
             var containerBuilder = new ContainerBuilder();
             containerBuilder.RegisterModule<ApplicationModule>();
+            containerBuilder.Register(c => Environment.GetFolderPath(Environment.SpecialFolder.Personal)).Named<string>(App.BootstrapParamaters.LocalAppDataFolder);
```

Comment:
Maybe I didn't explained it right..

>Why using the value from App.BootstrapParamaters.LocalAppDataFolder (which could be, /home/someUser/AppData)

That's a constant and I'm using it as parameter name. That const is defined at application level (platform-independent), as:

```
public static class BootstrapParamaters
{
      public const string LocalAppDataFolder = "LocalAppDataFolder";
}
```

> so the actual name of the registration does not need to be the actual contents. (In fact, I think it shouldn't).

Exactly, it's not.

> So, right now you'll have in your registrations table:
> Registration Name: /home/someUser/AppData, Value: /home/someUser/Personal
> I suggest we should have:
> Registration Name: ApplicationData, Value: /home/someUser/Personal

What you're suggesting is what we have right now.

Maybe the confusion came because that const was defined on the other PR. Do you think we can make it more obvious if we rename something?